### ISSUE TITLE
[Analytics] Launch scripts fetch resources from hard-coded domain

### ISSUE THREAD

**Issue by pahupe** [urfuwo/hlx-test#392]
Observation: Launch scripts (here: EDS News DEV) try to fetch resources from hard-coded domain www-qa.sap.com, which fails due to CORS issues. 

Needs to be raised with EPAM

Console error: 
Access to XMLHttpRequest at 'https://www-qa.sap.com/etc/designs/sapdx/clientlib-ip-detection/js/getCountryCode.json' from origin ... has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource

**Comment by pahupe** [comment]
Parked until Sankalp's return on Apr 2

**Comment by pahupe** [comment]
Has been resolved for QA, as the domain now matches
